### ISSUE TITLE
Node 18

### ISSUE THREAD

**Issue by galago** [strapi-community/strapi-provider-upload-google-cloud-storage#180]
After upgrading to latest version of Strapi that requires Node 18 I found an issue with this plugin

strapi-provider-upload-google-cloud-storage@4.0.0: The engine "node" is incompatible with this module. Expected version ">=12.x.x <=16.x.x". Got "18.16.1"

Is there any update coming to fix this issue?

**Comment by alumedot** [comment]
@galago, upgrade the library to v4.10.5

But the minimum node version must be updated anyway since v16 EOL

**Comment by galago** [comment]
yeah, it's updated to the latest version but now Strapi is in V18. So this module won't work anymore

**Comment by edporras** [comment]
+1 for this. If I enforce node >= 18 in my package.json, the build fails because of this plugin. As per the 4.14.4 release notes, strapi is no longer guaranteed to be compatible with node < 18:

https://github.com/strapi/strapi/releases/tag/v4.14.4

**Comment by galago** [comment]
any news about this?

**Comment by cyp3rius** [comment]
Closing, hence no activity & support only for Strapi 5.
